data/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Pls, if you not use the xmls:android , remove it

Reply:
Done!